docs/howtos/audit-scanner.md

```diff
@@ -64,3 +64,31 @@ scanner settings by changing the kubewarden-controller chart
 
 See [here](../explanations/audit-scanner) more information about the Audit
 Scanner.
+
+## Optional: Installation of Policy Reporter UI
+
+The `kubewarden-controller` chart comes with a subchart of the [Policy Reporter](https://kyverno.github.io/policy-reporter).
+It is disabled by default, and can be enabled by setting `auditScanner.policyReporter=true`.
+The values of the Policy Reporter subchart are exposed under the `policyReporter` key of
+the `kubewarden-controller` values.
```

Comment:
Maybe it would be nice to show a small yaml snippet of that:

```yaml
kubewarden-controller:
  policyReporter: true
```

Reply:
done in https://github.com/kubewarden/docs/pull/238/commits/8b7a7e1a0c79943fbac08f81cda3b6b55cd01ba8, which is indeed better.

Worth noting, been trying with minikube and the minikube ingress addon, and still I always get the ingress stuck in `Scheduled for sync` ... yet it seems correctly configured.